project/src/main/kotlin/org/ionproject/core/calendar/CalendarController.kt

```diff
@@ -0,0 +1,22 @@
+package org.ionproject.core.calendar
+
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/calendar", produces = [ "text/plain" ] )
```

Comment:
Add the text/calendar media type in the Media file and add a StringHttpMessageConverter to the CoreApplication file. This last step should be done after pulling from the branch feature/gh-30-json-home or master if the former branch is merged.